### DISCUSSION TITLE
Hello! G'day! Olá! Hallo! Bonjour! Ciao! Kia ora! Nǐ hǎo!

### DISCUSSION THREAD

**Discussion by mo-roger-milton** [MetOffice/simulation-systems#137] [Monsoon]
For those of you who have read the latest Monsoon2 newsletter, please say Hi! and tell us how convenient or not using GitHub discussions is for you. Thank you!

**Comment by dennissergeev** [comment]
Hi! I'm a long-term user of GitHub so as far as I'm concerned, using GitHub discussions is great! :+1:

**Reply by mo-roger-milton** [reply]
Thanks Denis! Good to know that someone both reads the newsletters and might find using GitHub discussion useful. Have a great weekend!

**Comment by mo-scottirvine** [comment]
"Hi!", first-time user of GitHub here.

**Reply by mo-roger-milton** [reply]
Hullo Scott! Glad you've found us!
